hack/dind-cluster.sh

```diff
@@ -324,12 +326,26 @@ os::provision::install-cmds ${DEPLOYED_ROOT}"
 }
 
 function nodes-are-ready() {
-  # Skip the SDN node since nothing is intended to be scheduled on it.
-  local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
-KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep -v ${SDN_NODE_NAME} | grep -v NotReady | grep Ready | wc -l")
-  node_count=$(echo "${node_count}" | tr -d '\r')
-  test "${node_count}" -ge "${NODE_COUNT}"
+  local bin_path=$(os::provision::get-bin-output-path "${ORIGIN_ROOT}")
+  local kc=$(os::provision::get-admin-config ${CONFIG_ROOT})
+  read -d '' template <<'EOF'
+{{range $item := .items}}
+  {{if not .spec.unschedulable}}
+    {{range .status.conditions}}
+      {{if eq .type "Ready"}}
+        {{if eq .status "True"}}
+          {{printf "%s\\n" $item.metadata.name}}
+        {{end}}
+      {{end}}
+    {{end}}
+  {{end}}
+{{end}}
+EOF
+  # Remove formatting before use
+  template=$(echo "${template}" | tr -d '\n' | sed -e 's/} \+/}/g')
```

Comment:
I have checked this to work locally and via the extended networking job that returned SUCCESS.  The embedded \n is escaped specifically for this reason.  I think it's cleaner to have to escape newlines than to have to have every line quoted and require line continuation.  I didn't strip formatting because of a compatibility issue, though, I'd just rather have commands be as succinct as possible to make it easier to deal with when debugging.